_data/simple-icons.json

```diff
@@ -7037,7 +7037,7 @@
         },
         {
             "title": "LGTM",
-            "hex": "FFFFFF",
+            "hex": "000000",
             "source": "https://lgtm.com"
         },
```

Comment:
Looks like we should remove this one for V10. It redirects to https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/

Reply:
Sure. We can remove those icons with another pull request.